ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java

```diff
@@ -75,20 +75,16 @@ public Bundle build() {
     return mBundle;
   }
 
-  public static AdConfig adConfigWithNetworkExtras(Bundle networkExtras) {
+  public static AdConfig adConfigWithNetworkExtras(Bundle networkExtras, boolean defaultMuteState) {
     AdConfig adConfig = new AdConfig();
-    adConfig.setMuted(false); // Unmute fullscreen ads by default
+    adConfig.setMuted(defaultMuteState);
+
     if (networkExtras != null) {
-      adConfig.setMuted(networkExtras.getBoolean(EXTRA_START_MUTED, false));
+      adConfig.setMuted(networkExtras.getBoolean(EXTRA_START_MUTED, defaultMuteState));
```

Comment:
@jiancchen would be good to test the default audio/mute behavior , along with the latest SDK if on integration theres no network extra added